pkg/shared/db-schemas/init.sql

```diff
@@ -170,6 +172,41 @@ FOR EACH ROW
 BEGIN
     UPDATE note
     SET updated_at = (strftime('%s', 'now') * 1000)
-    WHERE id = OLD.note_id;
+    WHERE id = OLD.note_id AND committed = 0;
+END;
+
+CREATE TABLE custom_item (
+	id INTEGER NOT NULL,
+	title TEXT,
+	price DECIMAL,
+	note_id INTEGER,
+	PRIMARY KEY (id, note_id)
+);
+
+CREATE TRIGGER update_note_timestamp_after_custom_item_insert
```

Comment:
Is this also affected by #650 / #647? If so, can still go in and be updated after

Reply:
Yes, it absolutely is: this was implemented before the updated at discussions/pinpointing of the sync test problem and is included in the DB timestamp related fix.

Furthermore, I have fixed that, partially, and have it in a WIP branch. The plan is to go over all of the affected functionality, apply the fix (imperative/explicit JS timestamp usage) and separate from the WIP branch in to a PR. The WIP branch: https://github.com/librocco/librocco/tree/db-refactor/wiring-in-wip